packages/service-library/src/servicelib/rabbitmq/_rpc_router.py

```diff
@@ -64,15 +64,17 @@ async def _wrapper(*args, **kwargs):
                         ):
                             raise
 
-                        _logger.exception("Unhandled exception:")
+                        _logger.exception(
+                            "Unhandled exception: %s", exc  # noqa: TRY401
+                        )
                         # NOTE: we do not return internal exceptions over RPC
                         formatted_traceback = "\n".join(
                             traceback.format_tb(exc.__traceback__)
                         )
                         raise RPCServerError(
                             method_name=func.__name__,
                             exc_type=f"{exc.__class__.__module__}.{exc.__class__.__name__}",
-                            msg=f"{formatted_traceback}",
+                            msg=f"{exc} {formatted_traceback}",
```

Comment:
please check here with @pcrespov, he changed that lately and you are putting this back in I think

Reply:
Will change it slightly to make it more readable.